### PR TITLE
cleanup unneeded basename call

### PR DESCRIPTION
**Pull request by yihuang**

### CODE REVIEW

File: bundle-linux.sh [hunk 1]
```diff
@@ -72,7 +72,7 @@ bundleExe() {
   local copied_exe="$out/$exe_dir/$exe_name"
   cp "$exe" "$copied_exe"
   chmod +w "$copied_exe"
-  patchelf --set-interpreter "$(basename "interpreter")" --set-rpath "\$ORIGIN/../$lib_dir" "$copied_exe"
+  patchelf --set-interpreter "interpreter" --set-rpath "\$ORIGIN/../$lib_dir" "$copied_exe"
```
Comment (yihuang) [inline bundle-linux.sh:75]: The purpose of setting it to `"interpreter"` is just to make it not runnable on it's own, but require the bin script to provide the correct interpreter, right?

Comment (3noch) [reply bundle-linux.sh:75]: We're not actually using this Interpreter, correct. The script overrides this.

Interpreter is likely a nix path here. Nuke refs would turn that path into a broken nix path. But I wanted to break it without making it obvious the file was built from nix.

**Comment by elliot-midnight** [comment]
I prefer not having a full nix path in the interpreter that gets nuked.

**Comment by 3noch** [comment]
Looks like #18 fixed this. I didn't realize I had made a mistake!

**Comment by yihuang** [comment]
> Looks like #18 fixed this. I didn't realize I had made a mistake!

I think it's not a mistake, you deliberately set interpreter to constant `"interpreter"` to make it runnable on it's own. But I guess it's alright either way.

**Comment by 3noch** [comment]
I actually didn't mean for it to be a constant. 😊

**Pull request closed without merging**